NSelene/SeleneElement.cs

```diff
@@ -366,5 +382,30 @@ public static SeleneCollection SS(this SeleneElement selement, string cssSelecto
                 return selement.FindAll(cssSelector);
             }
         }
+
+        public static class JSElementExtensions
+        {
+            public static SeleneElement JSScrollIntoView(this SeleneElement selement)
+            {
+                selement.ExecuteScript("element.scrollIntoView({ behavior: 'smooth', block: 'center'})");
+                return selement;
+            }
+
+            public static SeleneElement JSClickWithOffset(this SeleneElement selement, int offX, int offY)
```

Comment:
should be JsClickWithOffset
I also suggest to name params as xOffset and yOffset for better readability. There is a common guideline in naming to not use shortcuts, except the "popular" ones:) 
 
by the way... we could make it just simple JsClick(this SeleneElement element, int xOffset = 0, int yOffset = 0)
this will be more consistent with Selenide in Java...

from other point of view... JsClickWithOffset(100, 100) sounds more readable than JsClick(100, 100)
hm... but maybe JsClick(100, 100) is also ok...

Reply:
hm... i will create an issue to discuss this... let's proceed the conversation there...

Reply:
https://github.com/yashaka/NSelene/issues/51

Reply:
Agree, for now will just remove this method, at least we know now that we can build methods like that with this construct.